x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/dataframe/action/StopDataFrameTransformAction.java

```diff
@@ -56,15 +57,17 @@ public Response newResponse() {
         private final String id;
         private final boolean waitForCompletion;
         private final boolean force;
+        private final boolean allowNoResources;
```

Comment:
Rename `allowNoResources` to `allowNoMatch` throughout this file, including the getter and setter.